source/whats-new.txt

```diff
@@ -171,6 +172,37 @@ This release includes the following improvements and fixes:
 
 - The ``Collection.Clone()`` method does not return an error.
 
+.. _golang-version-1.17:
+
+What's New in 1.17
+------------------
+
+.. important::
+
+   v1.17 is the final planned v1.x version release. This version will
+   receive bug fixes, but future development and features will be
+   included only in 2.x versions of the driver.
+
+The 1.17 {+driver-short+} release includes the following improvements
+and fixes:
```

Comment:
Should we specifically mention that this minor release includes partial support for MongoDB 8.0 (with the exception of bulk write)?

Reply:
Yes, I can add here and in the compat table